test/db/tools/rz_asm

```diff
@@ -132,7 +132,7 @@ RUN
 
 NAME=rz-asm #1167 5
 FILE=-
-CMDS=!rz-asm -s att -a x86.as -b 64 'add $33, %rbx'
+CMDS=!rz-asm -s att -a x86.as -b 64 "add \\$33, %rbx"
```

Comment:
Honestly I think we should just have separate Windows and non-Windows tests i.e. 1 test in `db/archos/windows-x64/` and another test in `db/archos/non-windows-x64/`.

Reply:
Instead of using system as it is done now, I'm going to use the subprocess api, so we don't involve the system shell anymore. By doing this we should fix the problem on both linux and windows

Reply:
Oh ok